.github/workflows/build.yaml

```diff
@@ -33,11 +33,27 @@ jobs:
         run: |
           set -euo pipefail
           if [ '${{ inputs.mode }}' == 'release' ]; then
-            make verify-extended
+            mkdir -p /tmp/blobs.d
+            make verify-extended |& tee /tmp/blobs.d/tests-log.txt
+            tar czf /tmp/blobs.d/tests-log.tar.gz -C/tmp/blobs.d tests-log.txt
           else
              # tests are run using prow for non-release-runs, so early-exit
             exit 0
```

Comment:
Does `add-tests-report-to-component-descriptor` step still work when `/tmp/blobs.d/tests-log.tar.gz` does not exist? This could happen in the early exit case.

If the file must exist, we could create an empty file or a dummy file.

```
        run: |
          set -euo pipefail
          mkdir -p /tmp/blobs.d
          if [ '${{ inputs.mode }}' == 'release' ]; then
            make verify-extended |& tee /tmp/blobs.d/tests-log.txt
            tar czf /tmp/blobs.d/tests-log.tar.gz -C/tmp/blobs.d tests-log.txt
          else
             # create an empty file for add-tests-report-to-component-descriptor step
             touch /tmp/blobs.d/tests-log.tar.gz
             # tests are run using prow for non-release-runs, so early-exit
            exit 0
          fi
```